### DISCUSSION TITLE
Should we protect the main Branch of org.eclipse.mylyn?

### DISCUSSION THREAD

**Discussion by BeckerFrank** [eclipse-mylyn/.github#16] [Polls]
With the request to change the Default Branch I can also request that the main Branch is protected.
With a protected main Branch changes need a Pull Request.

Do you think we should activate this now?

**Comment by BeckerFrank** [comment]
What is the reason  why we should not protect the main Branch?

**Reply by ruspl-afed** [reply]
Good question. I'm not aware of any.

**Reply by wimjongman** [reply]
+1 thanks
